### ISSUE TITLE
Feature Request: support for history pushState and popState

### ISSUE THREAD

**Issue by scosist** [n33/jquery.poptrox#37]
I don't see any recent development happening on github but I've been experimenting with a 2.5.2-dev version of this plugin that came with the materialize theme.

Any chance you could add a setting for onPopupSwitch and expose $x.attr('src') in the context of onPopupSwitch  and onPopupOpen so that one could easily use history.pushState?

Related issue: in Firefox, when binding to the onClick event with pushState, of a link that triggers an iframe poptrox-popup, I was left with a blank iframe modal. Same result unexpectedly occurs in Firefox when triggering the click event of a link on popstate. Seems to work without a hitch in Chrome and Opera though.

**Comment by ajlkn** [comment]
Good idea. Rewriting Poptrox from scratch relatively soon so I'll look into this.
